dockstore-webservice/src/main/java/io/dockstore/webservice/core/Token.java

```diff
@@ -87,33 +87,40 @@
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "token_id_seq")
     @SequenceGenerator(name = "token_id_seq", sequenceName = "token_id_seq", allocationSize = 1)
-    @ApiModelProperty(value = "Implementation specific ID for the token in this web service", position = 0, readOnly = true)
```

Comment:
Are these positions being deleted on purpose?

Reply:
> minor questions
> 
> > However, when I tried to modify all the responses of the token endpoints to hide the token, the login functionality didn't work properly, so I left the response as is.
> 
> Is the response stored by the UI after login?

I don't think it's being stored, from the looks of https://github.com/dockstore/dockstore-ui2/blob/fdfe977105f2c49e5ef2ea564cb4140d31cb5a7a/src/app/loginComponents/auth/auth.component.ts#L85. Only the tokens retrieved using `GET /users/{userId}/tokens` are stored.



> Are these positions being deleted on purpose?

Yeah, I deleted them because some of the positions were conflicting in swagger.yaml since they had the same position number, so in swagger-ui, not all the fields showed up. In openapi.yaml, I noticed that it didn't show positions, so it didn't matter

Reply:
Ah, I see. `onlineProfileId` is new and should be moved back to not conflict (might be 7?)
But do restore the other positions to be safe, I think we put this in for a particular reason but I cannot remember why for this particular example.